searx/__init__.py

```diff
@@ -64,3 +64,49 @@
 if not searx_debug and settings['server']['secret_key'] == 'ultrasecretkey':
     logger.error('server.secret_key is not changed. Please use something else instead of ultrasecretkey.')
     exit(1)
+
+
+class _brand_namespace:
```

Comment:
Technically, this class shouldn't exist: everywhere where there is `brand.CONTACT_URL` it should be `settings['global'].get('contact_url', False)` ?

Or am I missing something ?

But it is convenient way to remove `searx/brand.py` with few changes.

Reply:
> Or am I missing something ?

no, you are correct.

> But it is convenient way to remove searx/brand.py with few changes.

This was my intention, we can remove the usage of brand-namepace afterwards.